tests/src/services/tiers.service.test.ts

```diff
@@ -304,17 +304,35 @@ describe('TiersService tests', () => {
   });
 
   describe('applyVpnFeatures()', () => {
-    it('When it is called, then it does not throw', async () => {
+    it('When VPN is enabled, then it adds the user tier to the VPN database', async () => {
```

Comment:
_then a request to enable user's tier on the VPN service is sent_

Note I am not specifying how that request is being sent (HTTP, gRPC, through a gateway, etc)